community/planning/griddle/openapi.yaml

```diff
@@ -0,0 +1,2146 @@
+            on top of Grid.
+          content:
+            application/json:
+              schema:
+                properties:
+                  link:
+                    $ref: "#/components/schemas/BatchPostResponse"
+        "400":
+          $ref: "#/components/responses/400BadRequest"
+        "500":
+          $ref: "#/components/responses/500ServerError"
+        "503":
+          $ref: "#/components/responses/503ServiceUnavailable"
+    delete:
+      tags:
+        - Locations
+      summary: Remove a Location
+      operationId: delete_location
+      parameters:
+        - name: location_id
+          in: path
+          description: ID of the location to delete
+          required: true
+          schema:
+            type: string
+        - $ref: "#/components/parameters/service_id"
+      responses:
+        "202":
+          description: |
+            The location was submitted for validation but has not yet been
+            evaluated. After the server receives the location for validation, it
+            will attempt to execute the transaction contained in the batch.
+            If the transaction is successful, the distributed ledger's
+            state will be updated and Grid will receive a state delta event from the
+            distributed ledger. The Grid Daemon processes this event and updates
+            the Grid database accordingly.
+            This means that there is a delay between when transactions are
+            submitted and when the new data is actually available to fetch using
+            the Grid REST API GET endpoints. This delay can range from a few
+            milliseconds to several minutes depending on the underlying ledger.
+            Developers should take this delay into account when designing apps
+            on top of Grid.
+          content:
+            application/json:
+              schema:
+                properties:
+                  link:
+                    $ref: "#/components/schemas/BatchPostResponse"
+        "400":
+          $ref: "#/components/responses/400BadRequest"
+        "500":
+          $ref: "#/components/responses/500ServerError"
+        "503":
+          $ref: "#/components/responses/503ServiceUnavailable"
+
+  # Pike
+  /organizations:
+    get:
+      tags:
+        - Pike
+      summary: Lists all organizations
+      operationId: list_organizations
+      parameters:
+        - $ref: "#/components/parameters/service_id"
+        - $ref: "#/components/parameters/page_offset"
+        - $ref: "#/components/parameters/page_limit"
+      responses:
+        "200":
+          description: |
+            Successful request. The response will include a JSON list of the
+            requested organizations.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/OrganizationList"
+        "400":
+          $ref: "#/components/responses/400BadRequest"
+        "500":
```

Comment:
It looks like above in the list_locations op we handle 404s. Do we not handle 404s here, or just missed ?

Reply:
404s added to get requests